From 8fa42aa9d86ca10e183ea776150a5ec2d7023bc5 Mon Sep 17 00:00:00 2001
From: Saurabh Sengar <saurabh.singh@xilinx.com>
Date: Wed, 9 May 2018 15:39:18 +0530
Subject: [PATCH 0354/1851] dt-bindings: display: xlnx: mixer: Adding
 xlnx,layer-primary property

commit ae98a105611f4d061499a0d2e635f1836328cb25 from
https://github.com/Xilinx/linux-xlnx.git

Adding documentation for xlnx,layer-primary property.
This property is required by driver but mistakenly left out in doc.

Signed-off-by: Saurabh Sengar <saurabh.singh@xilinx.com>
Reviewed-by: Vishal Sagar <vishal.sagar@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt b/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt
index 53f37e2b1a6d..94d43cd00345 100644
--- a/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt
+++ b/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt
@@ -25,6 +25,8 @@ Required properties:
    not required for overlay layers
  - xlnx,layer-max-height: max layer height, mandatory for master layer
    Not required for overlay layers
+ - xlnx,layer-primary: denotes the primary layer, should be mentioned in node
+   of layer which is expected to be constructing the primary plane
 
 Optional properties:
  - dmas: dma attach to layer, mandatory for master layer
@@ -92,6 +94,7 @@ Example:
 			dmas = <&axi_vdma_0 0>;
 			dma-names = "dma0";
 			xlnx,layer-streaming;
+			xlnx,layer-primary;
 		};
 		xv_mix_overlay_1: layer_1 {
 			xlnx,layer-id = <1>;
-- 
2.31.1

